ui-v2/app/routes/dc/acls/create.js

```diff
@@ -13,6 +13,8 @@ export default Route.extend(WithAclActions, {
   },
   model: function(params) {
     this.item = get(this, 'repo').create();
+    // TODO: Why didn't I have to do this for KV's?
+    set(this.item, 'Name', '');
```

Comment:
In the ember model definition you can add a `defaultValue` so you wouldn't have to do this here.

Reply:
Oh cool, yeah good call. Weirdly I never had to do this in KV's even though it use the same approach, didn't look in the model though, maybe I have something different there now you mention.